Configuration/TCA/tx_seminars_seminars.php~

```diff
@@ -0,0 +1,1130 @@
+<?php
+defined('TYPO3_MODE') or die();
```

Comment:
Please remove the backup files from this PR.

Reply:
done